src/polyfills.ts

```diff
@@ -55,7 +55,6 @@
 /***************************************************************************************************
  * Zone JS is required by default for Angular itself.
```

Comment:
Silly thing. We probably should just remove this comment as it sounds like it's no longer needed.

Reply:
Zone JS is still needed, I just moved it down and forgot to move this comment. However since it also has an inline comment, and because of the move, the division between Angular's own, and "Application Imports" isn't as clear cut anymore. I removed this comment